dev/MRTCore/packaging/MrtCore.PriGen.targets

```diff
@@ -7,6 +7,16 @@
     <TargetPlatformIdentifierAdjusted>UAP</TargetPlatformIdentifierAdjusted>
   </PropertyGroup>
 
+  <!--
```

Comment:
my impression is WinUI app doesn't import this file. Does everything still work for WinUI app?

Reply:
WinUI with WAP does import it:

<img width="953" alt="image" src="https://user-images.githubusercontent.com/35987549/162053854-2a51228a-a97d-43fa-ab66-874b92a588d1.png">

<img width="431" alt="image" src="https://user-images.githubusercontent.com/35987549/162054096-c95a744a-4afd-4ab1-a724-3e69a79061d5.png">

WinUI single project MSIX doesn't:

<img width="629" alt="image" src="https://user-images.githubusercontent.com/35987549/162055271-b0c6f951-52df-4e05-929d-c70993a54b33.png">

<img width="629" alt="image" src="https://user-images.githubusercontent.com/35987549/162054959-2c3c66f4-c721-4b05-9bad-7685fa1a9933.png">

These properties should be added to Microsoft.Build.Msix.Pri.targets for default file inclusion to work in single project MSIX projects (I just tested - it doesn't).

Default file inclusion works as expected in a WinUI WAP app, the PRI is correct too.

Unless you have objections, I'll also update Microsoft.Build.Msix.Pri.targets in this change.

Reply:
I will also add detailed notes to How Verified so it's clear what I tested

Reply:
Actually, Microsoft.Build.Msix.Pri.targets isn't in this repo. So that'll have to be a separate change.